style.css

```
@@ -0,0 +1,624 @@
+/*- Grijs `#7A7A7A` (voor de service-tegel tekst);*/
+
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 20px;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-family: Merriweather, Serif;
+}
+
+p {
+    font-family: Roboto, Serif;
+}
+
+li,
+a {
+    font-family: Roboto, Serif;
+    font-weight: normal;
+    cursor: pointer;
+}
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.outer-content-container {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+}
+
+.inner-content-container {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1400px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+main>.inner-content-container {
+    height: auto;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+#header {
+    background: hsla(241, 78%, 60%, 1);
+    background: linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    background: -moz-linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    background: -webkit-linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    filter: progid: DXImageTransform.Microsoft.gradient(startColorstr="#4b49e8", endColorstr="#5722cd", GradientType=1);
+    color: #F3F6F9;
+    flex: 1;
+    position: relative;
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 calc(100% - 50px));
+}
+
+#header, li a {
+    color: #F3F6F9;
+}
+
+.header-heading_navigation {
+    display: flex;
+    justify-content: flex-end;
+    padding-right: 30px;
+}
+
+.header-menu-list {
+    display: flex;
+    justify-content: space-between;
+    flex-basis: 400px;
+}
+
+.header-menu-list-item {
+    list-style: none;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.header-content_cta {
+    display: flex;
+    flex-direction: column;
+    padding: 10px 300px 200px 300px;
+    align-items: center;
+    text-align: center;
+}
+
+.header-content_cta > h2 {
+    font-weight: 300;
+    font-size: 1.2em;
+}
+
+.submit_button_header {
+    text-transform: uppercase;
+    background-color: #ED4385;
+    color: #F3F6F9;
+    border: none;
+    height: 40px;
+    width: 300px;
+    border-radius: 25px;
+    margin: 20px 0 20px 0;
```

Je zou dit nog korter kunnen schrijven:
```suggestion
    margin: 20px 0;
```